spec.md

```diff
@@ -724,8 +725,8 @@ If the plugin is unable to complete the ControllerPublishVolume call successfull
 If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
```

Comment:
This call is idempotent, too. Is there similar reasoning that can be applied that allows `OK` to be returned? Perhaps it's very SP-specific whether or not that is possible.

Reply:
In this case it likely isn't sufficient to check `volume_capability` as the `user_credentials` used to publish the volume to a node may impose different access policies.

Reply:
Perhaps
```
If the plugin determines that the volume has already been published to the node in a way that satisfies the current request it MAY respond with `0 OK`.
```

Reply:
I fix the wording and added a new error code. PTAL